DUNE_ND/submitLArRecoFNALJob.py

```diff
@@ -0,0 +1,240 @@
+# Script to submit a LArReco job to Fermigrid. Used by createLArRecoFNALJobs.py
+
+import os
+import platform
+import random
+import sys
+
+class parameters(object):
+
+    def __init__(self, label, sample, jobName, inFileName,
+                 startEvt, endEvt, runtime = '8h', memory = '2000MB'):
+
+        # Physics process label, e.g numu_cc_qe
+        self.label = label
+        # Sample label
+        self.sample = sample
+        # Job label
+        self.jobName = jobName
+        # Pandora input (simulation) file
+        self.inFileName = inFileName
+        # Start and end event numbers
+        self.startEvt = startEvt
+        self.endEvt = endEvt
+        # Number of events to process
+        self.NEvts = self.endEvt - self.startEvt
+
+        # Job runtime
+        self.runtime = runtime
+        # Job memory requirement (default = 2GB)
+        self.memory = memory
+        
+        # Home directory
+        self.home = os.getenv('HOME')
+        # Pandora release directory, assumed to be "$HOME/app/Pandora"
+        self.pandoraName = 'Pandora'
+        self.pandoraDir = '{0}/app/{1}'.format(self.home, self.pandoraName)
```

Comment:
Might be better to use MY_TEST_AREA here, as that should be fairly consistently defined.